### DISCUSSION TITLE
Remember taskbar screen

### DISCUSSION THREAD

**Discussion by terax6669** [valinet/ExplorerPatcher#1873] [Ideas]
Hi!

Every time the monitor configuration changes, the taskbar returns to the primary screen.

Would it be possible for EP to remember the screen where it was (perhaps similarily to how Windows is remembering window position, per monitor configuration) and put it there every time? I'm moving my taskbar every day, it's extremely tedious.

**Comment by pyrates999** [comment]
Explorer patcher could only do that if there was a setting that Microsoft provides for this.  So you most likely have to wait for Microsoft to fix this bug.
